client/components/common/ActiveUsers.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="active-users">
+    <v-avatar
+      v-tooltip="user.firstName + ' ' + user.lastName"
```

Comment:
This actually creates nice opportunity to use a filter:
```html
v-tooltip="user | fullName"
```
and later on in component:
```js
filters: {
  fullName: ({ firstName, lastName }) => `${firstName} ${lastName}`
}
```

Reply:
@vladimyr Good idea, will do it when first name and last name are available :)